flashinfer/mamba/ssu_incremental.py

```
@@ -0,0 +1,233 @@
+    if d_split is None:
+        # Auto-heuristic still clamped to 1 — re-enable as a separate
+        # tuning change once the benchmarking is in place.  The override
+        # knob is open and exercised by the d_split=2 correctness tests.
+        d_split = 1
+    assert d_split in (1, 2), (
+        f"d_split must be in {{1, 2}} for v12 (d_split=4 deferred), got {d_split}"
+    )
+    assert dim % d_split == 0, f"dim={dim} must be divisible by d_split={d_split}"
+    assert dim // d_split >= 32, (
+        f"d_split={d_split} gives D_PER_CTA={dim // d_split} < 32 "
+        "(output MMA m16n8 floor with _1×4 warp layout)"
```

Ruff (0.15.12)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)